services/ui-src/src/components/accordions/TemplateCardAccordion.tsx

```diff
@@ -4,18 +4,12 @@ import { AnyObject } from "types";
 import { parseCustomHtml } from "utils";
 
 export const TemplateCardAccordion = ({ verbiage, ...props }: Props) => (
-  <Accordion sx={sx.root} allowToggle={true} {...props}>
-    <AccordionItem sx={sx.text} label={verbiage.buttonLabel}>
+  <Accordion allowToggle={true} {...props}>
+    <AccordionItem label={verbiage.buttonLabel}>
       {parseCustomHtml(verbiage.text)}
-      {verbiage.table && (
-        <Table
-          content={verbiage.table}
-          variant="striped"
-          sxOverride={sx.table}
-        />
-      )}
+      {verbiage.table && <Table content={verbiage.table} variant="striped" />}
       {verbiage.list && (
-        <UnorderedList sx={sx.list}>
+        <UnorderedList variant="accordian">
```

Comment:
For this styling, do we want lists to share the same styling as everything under the accordion theme? Or does this application allow for only whats defined under list to be applied here?
Which is under 
```panel: 
        ...,
         list: {
            paddingLeft: "1rem",
          },```

Reply:
perfect catch! what was under panel wasn't working and I forgot to remove it, thank you for finding that!